src/AuthModule/Components/Login/Login.tsx

```diff
@@ -136,34 +132,33 @@ async function handleLogin(values:any) {
                     },
                   })}
                   type={!showPassword ? "text" : "password"}
-                  className="form-input form-control bg-transparent border-0 border border-bottom text-white p-1 pb-0 flex-grow-1"
-                  id="formControl"
+                  className="form-input form-control bg-transparent border-0 rounded-bottom-0  border border-bottom text-white p-1 pb-0 flex-grow-1"
                   placeholder="Enter your password"
                   aria-label="readonly input example"
                 />
-                {/* <span className="input-group-text border-0  bg-transparent position-absolute mt-4 end-0 p-2">
+                <span className="input-group-text border-0  bg-transparent position-absolute mt-4 end-0 p-2">
                   <i
```

Comment:
This should be a button and an icon inside it for better accessibility.  Currently, using the tap key, we will be able to access the inputs and submit buttons, the toggle functionality will not be accessible to people using the keyboard navigation through our website. Also, we should move the `onClick` handler to the button, not the icon.